go.mod

```diff
@@ -176,6 +176,3 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
-
-// ref: https://github.com/darccio/mergo/blob/2b1eb9c67d7332f286430af241180c5005a6a5a4/README.md?plain=1#L53
-replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.16
```

Comment:
I think this (mergo replace directive) was accidentally removed.

Reply:
We got rid of mergo, because it was not working as expected...

Reply:
It's currently an indirect dependency (`github.com/imdario/mergo v0.3.16 // indirect`), and the workaround says that in case of indirect dependency it may cause problems (hence the workaround):
> If the vanity URL is causing issues in your project due to a dependency pulling Mergo - it isn't a direct dependency in your project - it is recommended to use [replace](https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive) to pin the version to the last one with the old import URL: [...]

Let me know if I misunderstood something though.